src/main/java/marquez/resources/OwnerResource.java

```diff
@@ -31,6 +32,7 @@ public Response create(final Owner owner) {
 
   @Path("{owner}")
   @GET
+  @Produces(APPLICATION_JSON)
```

Comment:
Do we have a test to verify the response?

Reply:
We don't actually. Mind writing a test for the Owner resource?

Reply:
Ah so I changed it so that we will. We don't want to make an Owner fetchable?

```
HTTP/1.1 200 OK
Connection: close
Date: Tue, 28 Aug 2018 20:45:24 GMT
Content-Type: application/json
Vary: Accept-Encoding
Content-Length: 20

{"name":"analytics"}

Reply:
just mocks -- adding one for the response

Reply:
Looks like verifying a GET endpoint produces JSON is done via two separate tests:

1) a GET endpoint returns an expected object
2) the object can be serialized to JSON

Reply:
Agreed. Based on our discussions, performing `GET` on owner is not an op we want to support. When we introduce `/admin/*` that may be an op we'd like to support.

Also, the `OwnerTest` was in the `api` pkg. Assumed it be on `resources`. Might want to move things around, which is being addressed in #50